spacemesh/v2alpha1/smeshing_identities.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+import "google/api/visibility.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+package spacemesh.v2alpha1;
+
+message PoetServicesRequest {}
+
+enum IdentityStatus {
+  STATUS_UNSPECIFIED = 0;
+  STATUS_IS_SYNCING = 1;
+  STATUS_WAIT_FOR_POET_ROUND_START = 2;
+  STATUS_WAIT_FOR_POET_ROUND_END = 3;
+  STATUS_FETCHING_PROOFS = 4;
+  STATUS_POST_PROVING = 5;
+}
```

Comment:
The variants don't need to be prepended, it results in awkward variable names like `RegistrationStatus_STATUS_FAILED_REG`.
```suggestion
enum IdentityStatus {
  UNSPECIFIED = 0;
  IS_SYNCING = 1;
  WAIT_FOR_POET_ROUND_START = 2;
  WAIT_FOR_POET_ROUND_END = 3;
  FETCHING_PROOFS = 4;
  POST_PROVING = 5;
}
```

Reply:
The protobuf style guide recommends prefixing enums: https://protobuf.dev/programming-guides/style/#enums it leads to some awkward variable names, but without it it might break other code generators 🤷

Reply:
@poszu how should we handle this? Go for cleaner names for us or use the recommendation by protobuf style guide?

Reply:
I vote for cleaner names.

Reply:
@fasmat The guide recommends prefixing instead of enclosing enum in a `Message` because some languages don't support putting an enum in a struct. But here, the enum is at the root level, it's not inside another Message so I think it's fine.